website/docs/docs/dbt-versions/release-notes/74-Dec-2023/legacy-sl.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Deprecation: dbt Metrics and the legacy dbt Semantic Layer is now deprecated"
+description: "December 2023: For users on dbt v1.5 and lower, dbt Metrics and the legacy dbt Semantic Layer has been deprecated. Use the migration guide to migrate to and access the latest dbt Semantic Layer. "
+sidebar_label: "Deprecation: dbt Metrics and Legacy dbt Semantic Layer"
+sidebar_position: 09
+date: 2023-12-15
+---
+
+dbt Labs has deprecated dbt Metrics and the legacy dbt Semantic Layer, both supported on dbt version 1.5 or lower. These changes will be in effect on _December 15th, 2023_.
+
+This deprecation means dbt Metrics and the legacy Semantic Layer will no longer be supported. We will also remove the feature from the dbt Cloud user interface and documentation site.
```

Comment:
Is this release note intended to go out before or on the 15th? May want to consider changing the tense in this doc if it's going on the 15th to drive home that it's gone now
```suggestion
This deprecation means dbt Metrics and the legacy Semantic Layer are no longer supported. We also removed the feature from the dbt Cloud user interface and documentation site.
```